cmake/modules/hunter_dump_cmake_flags.cmake

```diff
@@ -20,6 +20,25 @@ function(hunter_dump_cmake_flags)
     hunter_internal_error("Unparsed arguments: ${x_UNPARSED_ARGUMENTS}")
   endif()
 
+
```

Comment:
> should I update PACKAGE_INTERNAL_DEPS_ID for all those packages as well?

There are more packages actually... This part is not well designed honestly. On practice most of these packages does't support iOS so it's not a problem.